ios/storage.md

```diff
@@ -113,14 +113,18 @@ let configuration = AWSServiceConfiguration(region: .USEast1, credentialsProvide
 let tuConf = AWSS3TransferUtilityConfiguration()
 tuConf.isAccelerateModeEnabled = true
 
-//Register a transfer utility object
+//Register a transfer utility object asynchronously
 AWSS3TransferUtility.register(
     with: configuration!,
     transferUtilityConfiguration: tuConf,
     forKey: "transfer-utility-with-advanced-options"
-)
+)  { (error) in
+     // Register complete, handle any errors. 
```

Comment:
This should be changed to 
"Registration was unsuccessful and encountered an error".